Sources/FluentKit/Properties/Field.swift

```diff
@@ -1,16 +1,10 @@
-extension Model {
-    public typealias Field<Value> = ModelField<Self, Value>
-        where Value: Codable
-}
-
-@propertyWrapper
-public final class ModelField<Model, Value>: AnyField, FieldRepresentable
-    where Model: FluentKit.Model, Value: Codable
-{
+@propertyWrapper @dynamicMemberLookup
+public final class ModelField<Model, Value>: AnyField, FieldRepresentable where Model: FieldGroup, Value: Codable {
+    public private(set) var keyPath = [String]()
```

Comment:
Could this be `keyPathPrefix` and we just store an optional prefix? We should also make sure it's initialized in the initializer, not with a default.

Reply:
Yeah, totally. Although an empty array storage is not a CoW class